aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -197,6 +199,10 @@ protected TableOperations newTableOps(TableIdentifier tableIdentifier) {
               IcebergToGlueConverter.getDatabaseName(tableIdentifier))
           .put(AwsProperties.LAKE_FORMATION_TABLE_NAME,
               IcebergToGlueConverter.getTableName(tableIdentifier))
+          .put(AwsProperties.DB_NAME,
```

Comment:
this is what I meant by we can reuse `s3.write.tags.` property, if we just add tags in that way we directly leverage existing logic.

Reply:
I have made the changes to leverage the existing logic.